bubblepicker/src/main/java/com/igalata/bubblepicker/physics/Engine.kt

```diff
@@ -17,10 +17,15 @@ object Engine : ContactListener {
     val selectedBodies: List<CircleBody>
         get() = bodies.filter { it.increased || it.toBeIncreased || it.isIncreasing }
     var maxSelectedCount: Int? = null
+
+    /**
+     * Represents what percentage of bigger screen dimension each bubble takes.
+     * Values 0..100 represent 15..35% of dimension
+     */
     var radius = 50
         set(value) {
             field = value
-            bubbleRadius = interpolate(0.1f, 0.25f, value / 100f)
+            bubbleRadius = interpolate(0.15f, 0.35f, value / 100f)
```

Comment:
Since I removed the exponential part, and on average our bubble picker was not perfect square, I checked on several different emulators and average ratio of height to width was 1.5, so I multiplied the min/max of interpolator to keep backward compatibility (so bubbles don't resize much suddently)